README.md

```diff
@@ -82,9 +84,13 @@ dansRoll();
 ```
 
 a. Where is closure used in this code? How can you tell?
+    - Closure is used when you are getting the "name" to console.log() their name and what they rolled. 
+
 b. Compare and contrast calling `dansRoll` the first and second time. What is always the same? What could change?
-c. What is the lexical scope of `newRoll`? 
+    - The name "Dan" will always be the same. The dice roll can possibly change. 
 
+c. What is the lexical scope of `newRoll`? 
+    - The lexical scope of 'newRoll' is the return function() within the personalDice() fucntion.
```

Comment:
I believe you have the right idea here but I'm going to re-word it just for clarification. 

Lexical scope is referring to the definition area of newRoll so in this case the lexcial scope is within the inner function that gets returned.